src/AeroSandbox/src/interceptors/storage/sql.ts

```
@@ -1,6 +1,6 @@
 import { type APIInterceptor, SupportEnum } from "$types/apiInterceptors.d.ts";
 
-import config from "$aero/examples/config";
+var config: any = {}
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)